ehitchhiking-ui/src/app/shared/enums/notification.types.ts

```diff
@@ -0,0 +1,5 @@
+export enum NotificationTypes {
+	success = 'Success!',
```

Comment:
property names should starts with capitalized letter

Reply:
done